src/test/java/uk/gov/hmcts/dts/fact/controllers/SearchControllerTest.java

```diff
@@ -75,6 +75,14 @@ void shouldSearchCourtsByPostcodeAndServiceArea() throws Exception {
         verify(courtService).getNearestCourtsByPostcodeSearch("OX1 1RZ", "Crime");
     }
 
+    @Test
+    void shouldSearchCourtsByPostcodeOnly() throws Exception {
+        mockMvc.perform(get(String.format("%s/%s", BASE_URL, "results/sw1a2by")))
+            .andExpect(status().isOk());
+
+        verify(courtService).getNearestCourtReferencesByPostcode("sw1a2by");
+    }
+
     @ParameterizedTest
```

Comment:
Since you have added spring boot pattern validation to the controller. Think we need a test here to test the case where the validation fails